includes/admin/class-wc-admin-post-types.php

```diff
@@ -627,6 +630,17 @@ public function bulk_edit_save( $post_id, $product ) {
 		$product->set_backorders( $backorders );
 
 		if ( 'yes' === get_option( 'woocommerce_manage_stock' ) ) {
+			$change_stock = absint( $_REQUEST['change_stock'] );
+			switch ( $change_stock ) {
+				case 2:
+					$stock_amount = wc_stock_amount( $product->get_stock_quantity() + $stock_amount );
+					break;
+				case 3:
+					$stock_amount = wc_stock_amount( $product->get_stock_quantity() - $stock_amount );
+					break;
+				default:
+					break;
+			}
 			$product->set_stock_quantity( $stock_amount );
```

Comment:
If you use `wc_update_product_stock`, you can ensure it's the correct value in case stock changes in another process.